mongo_connector/connector.py

```diff
@@ -304,6 +314,8 @@ def run(self):
         """Discovers the mongo cluster and creates a thread for each primary.
         """
         self.main_conn = self.create_authed_client()
+        global _min_source_version
+        _min_source_version = Version.from_client(self.main_conn)
```

Comment:
I need to update this logic to find the minimum MongoDB version in the cluster since the DocManager is shared. Is there a nice way to run a command on all nodes in a cluster with PyMongo?

Also, should we periodically update the version throughout the lifetime of mongo-connector in case the source cluster if upgraded or downgraded?